app/src/main/java/org/simple/clinic/newentry/PatientEntryScreenController.kt

```diff
@@ -368,20 +289,12 @@ class PatientEntryScreenController @Inject constructor(
   }
 
   private fun savePatient(events: Observable<UiEvent>): Observable<UiChange> {
-    val phoneNumberChanges = events.ofType<PatientPhoneNumberTextChanged>().map { it.phoneNumber }
-    val noPhoneToggles = events.ofType<PatientNoPhoneNumberToggled>().map { it.noneSelected }
     val colonyChanges = events.ofType<PatientColonyOrVillageTextChanged>().map { it.colonyOrVillage }
-    val noColonyToggles = events.ofType<PatientNoColonyOrVillageToggled>().map { it.noneSelected }
-
-    val isPhoneNumberValid = events
-        .ofType<PatientEntrySaveClicked>()
-        .withLatestFrom(phoneNumberChanges, noPhoneToggles)
-        .map { (_, phoneNumber, isNoneSelected) -> phoneNumber.isNotBlank() || isNoneSelected }
 
     val isColonyValid = events
```

Comment:
This validation can also be removed. It's handled by `OngoingPatientEntry#validationErrors()`.